descriptors/ModuleDescriptor-template.json

```diff
@@ -69,7 +69,7 @@
     },
     {
       "id": "users",
-      "version": "15.2"
+      "version": "15.2 16.0"
```

Comment:
OK. Have you conducted a real test for mod-user-import against mod-users with a more strict spec? Will it mean that mod-user-import now *also* offers a breaking change.. Eg what worked earlier (user import that is), may now fail??

Reply:
I realize when reading MODUSERS-212 that mod-users choked always when this was not set.

Reply:
It should not, no.  What we've changed here is making the addresstypeId field mandatory in the RAML.  However, it was already effectively required, since if you did not supply it, mod-users would throw a 500 error.  The only behavior that's actually changed here is what kind of error the module throws if an addresstypeId is not provided-before it was a generic 500 error, now you'd get a 400 with an explanation in the body.

Reply:
> I realize when reading MODUSERS-212 that mod-users choked always when this was not set. So no need to test further I guess.

Yup, and that is why we made it required in the raml.  It made no sense to us that it would be required by the logic but not be marked as required in the RAML.  The downside to that is that it's technically a breaking change that requires all dependent modules be updated.